docs/toc/processor.rst

```diff
@@ -0,0 +1,95 @@
+Processor
+==========
+
+A pipeline component that wraps inference model and set up inference related work.
+
+
+
+Usage
+------
+
+* wrap model by setting `self.model`
```

Comment:
There is no requirement to use `self.model`, this is all up to the implementation.